docs/getting-started/using-the-block-explorer.md

```diff
@@ -4,59 +4,69 @@
-`Transactions` | The number of transactions included in the block.
-`Votes`        | The number of proof-of-stake votes included in the block.
-`Fresh Stake`  | The number of new tickets purchased in this block.
-`Size`         | The size (in bytes) of the block.
+Each block is packed with information. There are two main sections inside a
+block: 
 
-Under `Latest Transactions`, you can see the transaction ID (txid) and
-the value (in DCR) transmitted across the network.
+1. **The Block Header**
+    - This shows summary information about this specific block, as well as a
+    link to the previous block in the chain.
+2. **Transactions**
+    - Consisting of:
+        - The Proof-of-Work Reward
+        - The Treasury Reward
+        - Proof-of-Stake Votes and Rewards
+        - Tickets which will enter the stakepool
+        - All other transactions about how DCR is transmitted across the network
 
 ---
 
-## ![](../img/dcr-icons/Blocks.svg){ .dcr-icon } Blocks
+## Block Headers
 
-Blocks can be found by searching for their block height number,
-clicking on a `Height` value from the home page, or from their
-`BlockHash` value. Older blocks will have lower block numbers. The top
-half of a block overview shows relevant information about this
-specific block. This information includes: the block height, the block
-hash, and several key network parameters, described below:
+The following table should help explain what you'll see in the top section of
+[dcrdata](https://dcrdata.decred.org). This is generally all information that's
+also found inside a block header, although it adds some additional context.
 
-Option                   | Explanation
+Parameter                | Explanation
 ---                      | ---
-`Number of Transactions` | The number of standard transactions (DCR sent from one user to another).
+`Block Hash`             | The unique hash of this specific block, which was found by the miner.
+`Confirmations`          | The number of new blocks which follow this block in the chain.
+`Size`                   | The size of the block (in bytes).
 `Height`                 | The height of the blockchain in which this block resides.
+`Version`                | The version of the block header.
+`Merkle Root`            | A hash value of all the transaction hashes and stake hashes (ticket purchases, votes, and revocations) included in this block.
+`Stake Root`             | Originally, this was a hash value of all the stake related transaction hashes in this block. However, as of [DCP-0005](https://github.com/decred/dcps/blob/master/dcp-0005/dcp-0005.mediawiki), the stake hashes were moved into the Merkle Root, and the Stake Root was repurposed to house header commitments. These includes additional proofs and compact filters, which enable a more secure SPV mode for lightweight clients.
+`Number of Transactions` | The number of standard transactions (DCR sent from one user to another).
```

Comment:
Please update this to "regular" as commented above. "Standard" has a slightly different meaning.

To give you some reference for these terms, [standard](https://github.com/decred/dcrd/blob/master/txscript/stdscript/README.md) and [regular](https://docs.decred.org/glossary/#regular-transaction-tree)